lib/bpk-component-flare/src/BpkFlare.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React, { type ElementProps, type Node } from 'react';
+import PropTypes from 'prop-types';
+import { StyleSheet, View, ViewPropTypes } from 'react-native';
+
+type ViewProps = ElementProps<typeof View>;
+type ViewStyleProp = $PropertyType<ViewProps, 'style'>;
+
+const styles = StyleSheet.create({
+  base: {
+    flex: 1,
+  },
+});
+
+export const INSET_PADDING_MODES = {
+  none: 'none',
+  bottom: 'bottom',
+};
+
+export type Props = {
+  children: Node,
+  insetPaddingMode: $Keys<typeof INSET_PADDING_MODES>,
+  style: ViewStyleProp,
+};
+
+const BpkFlare = (props: Props) => {
+  const { children, insetPaddingMode, style: userStyle, ...rest } = props;
+
+  const style = [styles.base];
+  if (userStyle) {
+    style.push(userStyle);
+  }
+
+  return (
+    <View style={style} {...rest}>
```

Comment:
Can we replace the view here with a `NativeFlare` component? The files would be `NativeFlare.ios` and `NativeFlare.android`  and they can just return a view and spread the props for now,

Reply:
Sure, will do that